src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -530,6 +529,7 @@ static inline int MPIDI_OFI_do_get(void *origin_addr,
                                         origin_bytes, origin_true_lb);
     MPIDI_Datatype_check_contig_size_lb(target_datatype, target_count, target_contig,
                                         target_bytes, target_true_lb);
+    MPIR_ERR_CHKANDJUMP((origin_bytes != target_bytes), mpi_errno, MPI_ERR_SIZE, "**rmasize");
 
```

Comment:
Why is it fixed only for `do_get` ?

Reply:
`do_put` did not need to fixed. I should modify the commit message saying that only `do_get` is corrected.